network/datadog_checks/network/config_models/instance.py

```diff
@@ -0,0 +1,57 @@
+# (C) Datadog, Inc. 2021-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from __future__ import annotations
+
+from typing import Optional, Sequence
+
+from pydantic import BaseModel, root_validator, validator
+
+from datadog_checks.base.utils.functions import identity
+from datadog_checks.base.utils.models import validation
+
+from . import defaults, validators
+
+
+class InstanceConfig(BaseModel):
+    class Config:
+        allow_mutation = False
+
+    blacklist_conntrack_metrics: Optional[Sequence[str]]
+    collect_aws_ena_metrics: Optional[bool]
+    collect_connection_queues: Optional[bool]
+    collect_connection_state: bool
```

Comment:
It doesn't look to be actually required https://github.com/DataDog/integrations-core/blob/f49996735d1973e35d66c1d4ea41bd4adc92d3fb/network/datadog_checks/network/network.py#L89